images/envoy-ratelimit/README.md

```diff
@@ -0,0 +1,31 @@
+<!--monopod:start-->
+# envoy-ratelimit
+| | |
+| - | - |
+| **Status** | experimental |
```

Comment:
We should aim to make this stable from the get-go

Reply:
What do you mean by `get-go`? 🤔

Reply:
Sorry for the miscommunication. I mean we should make sure this image is tested throughly (I think it is now, with the helm test) so that we can make it `stable` right away.

Reply:
Thanks! Now stabilized.